static/js/utils.js

```diff
@@ -17,7 +17,7 @@ function alert(type, message, delay) {
 
 window.onload = _ => {
   const menu = document.getElementById('menu')
-  const avatar = document.getElementById('avatarMenu')
+  const avatar = document.getElementById('avatar-img')
```

Comment:
Are you sure? https://github.com/neomad-team/neomad.org/blob/master/templates/layout.html#L26